pip_executable_choco/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,89 @@
+﻿function Get-RegistryValue($key, $value) {
+  $item = (Get-ItemProperty $key $value -ErrorAction SilentlyContinue)
+  if ($item -ne $null) { return $item.$value } else { return $null }
+}  
+
+function Get-Python-Home() {
```

Comment:
This should be called `Get-PythonHome`. There is typically only 1 dash separating the first verb and the noun.